Makefile

```diff
@@ -0,0 +1,30 @@
+unit:
+	@echo "Running unit tests in docker (heroku-18)..."
+	@docker run -v $(shell pwd):/buildpack:ro --rm -it -e "STACK=heroku-18" -e "BUILDPACK_HOME=/buildpack" heroku/heroku:18 bash -c 'cp -r /buildpack /buildpack_test; cd /buildpack_test/; test/unit;'
+	@echo ""
+
+v3:
+	@echo "Running v3 integration tests in docker (heroku-18)..."
+	@docker run -v $(shell pwd):/buildpack:ro --rm -it -e "STACK=heroku-18" -e "BUILDPACK_HOME=/buildpack" heroku/heroku:18 bash -c 'cp -r /buildpack /buildpack_test; cd /buildpack_test/; test/v3;'
```

Comment:
Should we be using `CNB_STACK_ID` instead of `STACK` for v3?

Reply:
probably. at least `CNB_STACK_ID` maybe both

Reply:
hmm, setting both when running v3 messes up the `bin/detect`, which checks for `CNB_STACK_ID`

Reply:
setting CNB_STACK_ID for v3 only and STACK for v2 now